less/main.less

```diff
@@ -0,0 +1,173 @@
+.title-container {
+    display: inline-flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    
+
+   .main-img {
+   display: flex;
+   align-items: center;
+   justify-content: center;
+   }
+
+    img{
+        max-width: 80%;
+          
+    }
+
+
+}
+//Media for title containeer
+@media (max-width: 500px) {
```

Comment:
nice job leaving comments for yourself! and nice media queries!
Comments are always super useful when coding (: